modules/Blockchain/Polygon/migrations/2_total_migration.js

```diff
@@ -161,6 +161,101 @@ module.exports = async (deployer, network, accounts) => {
             await hub.setContractAddress('Holding', holding.address);
             console.log('Updated contract in Hub contract map!');
         }
+        break;
+    case 'mainnet':
+        await deployer.deploy(Hub, { gas: 6000000, from: accounts[0] })
```

Comment:
why are we doing deployment of Hub two times?

Reply:
Each case is for different environment. When we start mainnet migration only one hub contract will be deployed

Reply:
Sorry, my error. I just found that there are two contracts for the same environment :)